src/lib/types/update-query.ts

```diff
@@ -1,5 +1,5 @@
 export interface UpdateQuery {
-	$set?: { [key: string]: any };
+	$set?: { [key: string]: { $ifNotExists: any } | any };
```

Comment:
Ideally we expand this interface with all possible operators. But that feels like a future improvement 👍 .